test/unit_tests/braket/experimental/autoqasm/test_program.py

```diff
@@ -87,36 +87,32 @@ def circuit(angle: float):
 
     @aq.main
     def zne(scale: int, angle: float) -> aq.BitVar:
-        for i in aq.range(scale):
+        if scale:
```

Comment:
Does it break if you change it back? This changes the behavior of this test and the program

Reply:
Alternatively, we can use the meta-programming style for this

Reply:
Yes, even with the old explicit syntax, FreeParameters inside an aq range breaks. For this test, I'll flip back to meta-programming and I'll create an issue for the bug

Reply:
https://github.com/orgs/amazon-braket/projects/2/views/1?pane=issue&itemId=45876614